test/dynamo/test_export.py

```diff
@@ -607,6 +677,63 @@ def func():
 
         dynamo_result = out_graph()
         self.assertTrue(torch._dynamo.utils.same(real_result, dynamo_result))
+    
+    def test_export_no_tensor_computation_with_aten_graph(self):
+        inp = [torch.randn(3)]
+        inp2 = 2
+        inps = [inp, inp2]
+
+        def func(x, y):
+            return x
+
+        opt_func = torch._dynamo.optimize("eager", nopython=True, dynamic=True)(func)
+        real_result = opt_func(*inps)
+
+        torch._dynamo.reset()
+
+        exported = torch._dynamo.export(func, aten_graph=True)(*inps)
+        out_graph = exported[0]
+
+        dynamo_result = out_graph(*inps)
+
+        self.assertTrue(torch._dynamo.utils.same(real_result, dynamo_result)) 
+        self.assertExpectedInline(
+                    out_graph.code.strip(),
+            """\
+def forward(self, x, y):
+    arg0, arg1, = fx_pytree.tree_flatten_spec(([x, y], {}), self._in_spec)
+    arg0_1 = arg0
+    _tensor_constant0 = self._tensor_constant0
```

Comment:
This is a little suspicious, but at least it's not being used in the result

Reply:
The _tensor_constant is created [here](https://github.com/pytorch/pytorch/blob/eeef68671d4f19d2f7d364ec6abb21d3c2696fad/torch/fx/_symbolic_trace.py#L822) when creating the aten graph. Is it reasonable?

Reply:
I made another minor modification to the code, we actually do not need to output anything for our graph, we just need to output ().

Since there's no tensor computation, any returned tensor can only come from the input. In that case, rewrite_signature will take care of adding it to the final graph output by looking at `result_traced`.

Reply:
nice!